### PR TITLE
[VUMM-653] Modify MergeTest.java to run in each PR

### PR DESCRIPTION
**Pull request by anandjakhaniya**
## Testing
- [ ] Unit test
- [ ] Deployed to dev env
- [ ] Other (explain) 

## Reverting
- [ ] Contains Migration - _Do Not Revert_

### CODE REVIEW

**Comment by anandjakhaniya** [comment]
@jkwatson-verta

File: backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ private static boolean isMigration() {
             Optional.ofNullable(System.getenv(CommonConstants.ENABLE_LIQUIBASE_MIGRATION_ENV_VAR))
                 .orElse("false"));
     LOGGER.info("ENABLE_LIQUIBASE_MIGRATION_ENV_VAR: {}", enableLiquibaseMigrationEnvVar);
-    return enableLiquibaseMigrationEnvVar;
+    return true;
```
Comment (jkwatson-verta) [inline backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java]: ahem

Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/configuration/EnabledMigration.java]: reverted

**Comment by anandjakhaniya** [comment]
@jkwatson-verta .

**Review by eheinlein-verta — APPROVED** [review]

**Review by jkwatson-verta — APPROVED** [review]

**Pull request merged**